auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -310,13 +328,19 @@ export default defineComponent({
       return affiliationInviteInfos.length > 0 && affiliationInviteInfos[0].status
     }
 
+    // Method to format dates
+    const formatDate = (dateString) => {
+      return moment(dateString).format('MMMM D, YYYY') // Format like "May 5, 2024"
```

Comment:
Does this fail safely if dateString is null/undefined/empty?

Reply:
Will update this.

Reply:
Updated, and show the date + time